### PR TITLE
Display primary nationality

### PR DESCRIPTION
**Pull request by Ayedrian**
The way the emojis are resized is not ideal with some rather ugly inline CSS "hacks", other than that displaying nationality in both the small card as well as the detailed view is implemented.

### CODE REVIEW

File: src/app/dashboard/person-preview/person-preview.component.html
Comment (Ayedrian) [inline src/app/dashboard/person-preview/person-preview.component.html:1]: Removing the badge class seems like a "necessary evil" to get the text displayed to the right of the badges again (previously there was a bug that the text floated onto the next line below, which caused it to render *behind* content that was actually supposed to be on the next line). Removing the badge class oddly fixes this behavior, but the gray background associated with badges is of course gone.

**Review by Mtze — CHANGES_REQUESTED** [review]
Ideally you implement a hover tooltip which shows the Countrycode of the flag in the overview page (I case somebody does not recognise the flag) 

In the detail view you should definitely display the Country Code next to the flag

**Comment by Ayedrian** [comment]
What changed: A new attribute in the CSV (a person's primary nationality, specified using ISO 3166-1 alpha-3) is displayed using the corresponding flag (emoji) in the preview card and using the flag as well as country code in the detailed person view.

Why the changes: A functional requirement was requested that instructors be able to see a person's primary nationality.

Testing steps: Start up a container using the README instructions, distribute the people in the pool according to some constraints of your choice, confirm that the emoji flags are being displayed in the preview cards and that hovering over them shows the country code, click on a person's card to open the detailed view and confirm that both the flag as well as the country code are listed.

**Review by Mtze — APPROVED** [review]
Changes seem to work with the example data - As soon as we have a working csv import again we can test it with real data

**Pull request merged**